### ISSUE TITLE
allow users to schedule offers

### ISSUE THREAD

**Issue by shortwavesurfer2009** [haveno-dex/haveno#1074]
Allow users to schedule offers in their local time such that the offer is automatically enabled at the beginning time and automatically disabled at the ending time. I figure for now this could apply to all offers at once. I could see where using this feature might lead to a small privacy leak due to having the schedule turning on during normal business hours in your country, which may narrow your possible location to a region. As an example if I live in the United States and I have my offer automatically turn on at 6 a.m. New York time and turn off at say 10 p.m. New York time then that might give away that I am a United States person.

**Comment by monerobull** [comment]
Nah, it is 1 button to activate and de-activate your offers. If you can't even do that, why would you be there to actually react to an offer?

**Comment by shortwavesurfer2009** [comment]
#1075 would require this

**Comment by monerobull** [comment]
But why? I'd argue it leads to less failed trades when you manually have to enable the offers in the morning. Automatic disable time makes sense in case you forget to turn offers off in the evening but automatic activation is bad. If you are there to respond to offers, you should be able to press 1 button to activate them, especially if you want to halve the trade time.

**Comment by shortwavesurfer2009** [comment]
Actually, that is a very good point.

**Comment by triskovac-zasmat-0g** [comment]
Without some kind of mobile notifications (push / mail / telegram) this is nonsense.

**Comment by shortwavesurfer2009** [comment]
1.0.12 now has trade sounds to start off with and this feature appears to have been added too
